src/common/personalizedDigest.ts

```diff
@@ -420,7 +420,7 @@ export const schedulePersonalizedDigestSubscriptions = async ({
 }: {
   queryBuilder: SelectQueryBuilder<UserPersonalizedDigest>;
   logger: FastifyBaseLogger;
-  sendType: UserPersonalizedDigestSendType;
+  sendType: UserPersonalizedDigestSendType[];
```

Comment:
just changing type so that we can support multiple sendTypes, for this function is just used for logging, so nothing breaking